packages/@sanity/default-layout/src/components/DefaultLayout.js

```diff
@@ -61,6 +72,16 @@ export default withRouterHOC(class DefaultLayout extends React.Component {
     })
   }
 
+  setLoadingScreenElement = element => {
+    if (element && this.state.showLoadingScreen) {
+      element.addEventListener('animationend', event => {
```

Comment:
This method should be assigned on the class instance, and should have a `componentWillUnmount` method that removes the event listener.

Reply:
👍